book/07-git-tools/sections/advanced-merging.asc

```diff
@@ -12,17 +12,17 @@ Samakatuwid, kung ikaw ay naghintay nang sobrang tagal upang i-merge ang dalawan
 Sa seksyon na ito, kami ay magpapatuloy kung ano ang mga maaaring isyu at anong mga kasangkapan na nagbibigay tulong upang panghawakan ang mga ito na mas madayang mga sitwasyon.
 At saka tatalakayin din natin ang iba't iba, na hindi pamantayan na mga uri ng mga merge na maaari mong gawin, pati na rin makita kung paano umatras sa mga merge na iyong nagawa.
 
-==== Merge Conflicts
+==== Pag-merge sa mga Kasalungat
 
-While we covered some basics on resolving merge conflicts in <<_git_branching#_basic_merge_conflicts>>, for more complex conflicts, Git provides a few tools to help you figure out what's going on and how to better deal with the conflict.
+Habang sakop namin ang maraming mga pangunahin na paglulutas ng merge na mga salungat sa <<_git_branching#_basic_merge_conflicts>>, para sa higit na kumplikado na mga salungat, Ang Git ay nagbibigay na ilang mga kasankapan para tulungan ka na tingnan kung ano ang nangyayari at kung paano mas mahusay na makitungo sa mga salungat.
```

Comment:
- `Habang sakop namin ang maraming mga pangunahin na paglulutas` to `Habang nasakop natin ang maraming mga batayan sa paglulutas`
- `kasankapan` to `kasangkapan`

Reply:
done change